solutions/cf-1536C.mdx

```diff
@@ -0,0 +1,145 @@
+---
+id: cf-1427G
+source: CF
+title: Diluc and Kaeya
+author: Kevin Sheng
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/91520)
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <vector>
+#include <unordered_map>
+#include <algorithm>
+
+using std::cout;
+using std::endl;
+using std::vector;
+
+// https://codeforces.com/problemset/problem/1536/C (input ommited due to length)
+int main() {
+	int test_num;
+	std::cin >> test_num;
+	for (int t = 0; t < test_num; t++) {
+		int len;
+		std::cin >> len;
+		vector<char> str(len);
```

Comment:
why can't you just use a string instead of a char vector?

Reply:
don't wanna put an extra #include

Reply:
I think the code would look cleaner and more straightforward with a string, but up to you

On an unrelated note, I think that strings usually do work without `#include <string>` but this is probably also implementation dependent